Options.lua

```diff
@@ -143,6 +143,17 @@ local optionProcessor =
 		end
 		return a_Idx + 2
 	end,
+	
+	-- -sy <pattern> -- Loads all symbols matching the pattern into the sandbox.
+	["-sy"] = function (a_Args, a_Idx, a_Options)
```

Comment:
How about `-f`, as Fill API?
Also, this file has a description at the top which needs to be updated.